Shared/shared.template.json

```diff
@@ -301,7 +459,7 @@
       "copy": {
         "name": "databasecopy",
         "count": "[length(parameters('databaseNames'))]",
-        "mode": "Serial"
+        "mode": "Parallel"
```

Comment:
Is this tested? I dont remember if there was a reason for running in Serial. The first thing that springs to mind is locks.

Reply:
It seems to run OK in dev

Reply:
I had a chat with @EwanNoble and he thought it may be something to do with using Elastic Pools. I tested using an ARM template that created a new SQL Server, Elastic Pool and two databases and it deployed OK.

Reply:
Whats the reason behind the change?

Reply:
It reduces the time taken to deploy the databases and therefore the whole inf release.

Reply:
Gut feel is that we are forgetting something but only one way to find out now.